test/rgb2hex.test.js

```diff
@@ -124,5 +124,11 @@ describe('rgb2hex should', () => {
             expect(rgb2hex('0px0px8pxrgb(0,0,0)').hex).toEqual(rgb2hex('rgb(0,0,0)').hex)
             expect(rgb2hex('solid2pxrgb(0,0,0)').hex).toEqual(rgb2hex('rgb(0,0,0)').hex)
         })
+
+        it('stuff that is prepended and appended', () => {
+            const url = 'https://bb6dcf512b33241ef106-b12771d2b2eb9c49cc5bd30e7bd6a290.ssl.cf2.rackcdn.com/5q2ppv8j1dld9di26je03y9k77arp45b'
```

Comment:
Can you update this url so that it doesn't point to my company's logo? 👼 🙏

Reply:
sure thing, my apologies!

Reply:
Just updated the example which now also includes the alpha channel :)
Thank you for your comments!